utility/README.md

```diff
@@ -1,679 +1,1309 @@
+  GetRelayChains() []RelayChain # The flavor(s) of Web3 hosted by this Application
+  GetGeoZone() GeoZone          # The physical geo-location identifier this Application registered in
+  GetNumServicers() uint8        # The number of Servicers requested per session
 }
 ```
 
+#### 3.4.2 Parameter Updates
 
-A Fisherman is able to modify their initial staking values including GeoZone, ServiceURL, and StakeAmount by submitting a StakeMsg while already staked. It is important to note that contrary to ServiceNodes, Fishermen GeoZone changes must be permissioned through the DAO ACL. In addition to that restriction, a StakeAmount is only able to be modified greater than or equal to the current value. This allows the protocol to not have to track pieces of stake from any Fisherman and enables an overall simpler implementation. 
-
+An Application can update any of the values in its on-chain attributes by submitting another `ApplicationStakeMsg` while it is already staked. Any parameter changes are permissible as long as the `StakeAmount` is equal to or greater than its current value.
 
-```
-# FishermenNode(Edit)StakeMsg Interface
-type FishermenStakeMsg interface {
-  GetPublicKey()  PublicKey       # identity of edited Fishermen
-  GetStakeAmount() BigInt         # must be greater than or equal to the current value
-  GetServiceURL() ServiceURL      # may be modified
-  GetGeoZone() LocationIdentifier # may be modified
-  GetOperatorPubKey() PublicKey   # may be modified
+```go
+type ApplicationStakeMsg interface {
+  GetPublicKey()  PublicKey     # The public cryptographic ID of the Fisherman
+  GetStakeAmount() BigInt       # May be modified to a value greater or equal to the current value
+  GetRelayChains() []RelayChain # May be modified
+  GetGeoZone() GeoZone          # May be modified
+  GetNumServicers() int8        # May be modified
 }
 ```
 
+#### 3.4.3 Unstaking
 
-A Fishermen is able to submit an UnstakeMsg to exit the network and remove themself from service. After a successful UnstakeMsg, the Fishermen is no longer eligible to receive monitoring requests. It is important to note, a Fisherman is permissioned to unstake if removed from the DAO ACL parameter. After FishermenUnstakingTime elapses, any Stake amount left is returned to the custodial account.
+An Application is able to submit an `UnstakeMsg` to exit and remove itself from the network. After a successful UnstakeMsg, the Application is no longer eligible to consume Web3 traffic from Servicers. After the `ApplicationUnstakingTime` unbonding time elapses, the remaining is returned to the custodial account.
 
+#### 3.4.4 Stake Burning
 
-```
-# FishermenUnstakeMsg Interface
-type FishermenUnstake interface {
-  GetAddress()  Address       # The address of the Fishermen unstaking
-}
-```
+Application stake burn is a necessary mechanism to ensure economic equilibrium at network maturity by balancing POKT inflation and deflation.
```

Comment:
How are they handled through gateway model? Didn’t see any details about whether the gateway security deposit is burned or if it’s burned through the app. I assume the app.